telemetry/telemetry.proto

```diff
@@ -4,6 +4,72 @@ import 'nanopb.proto';
 
 option java_outer_classname="TelemetryProto";
 
-message Telemetry {
-  optional int32 timestamp_ms = 1;
+//A DeviceInfoMessage should only be sent if a request is received from the telemetry viewer (ex. if the
+//		viewer sees a device it hasn't registered yet)
+//Otherwise, the viewer will just reference its config files (stored offline) for this information
+//Note: formerly NameIDMessage
+message DeviceInfoMessage { //send the device ID with its corresponding information
+	enum READING_TYPE {
+		VOLTAGE = 1;
+		CURRENT = 2;
+		TEMPERATURE = 3;
+		MCC = 4;
+		DRIVER_CONTROL = 5; //i.e. pedal board
+		OTHER = 127;
```

Comment:
True, I can't really see a reason why we'd ever need to send a measurement that couldn't have just been added to the enum listing